app/bootstrap.php

```diff
@@ -31,11 +31,9 @@
 		'line' => $e->getLine(),
 		'stack_trace' => $e->getTraceAsString()
 	] );
-	return new JsonResponse(
```

Comment:
I'll be merging this but will be providing a patch that checks the request for acceptable content types and returns either HTML or JSON.